packages/core/src/auth/i18n/fr.json

```diff
@@ -9,15 +9,14 @@
   "Auth_CannotOpenFile": "Impossible d'ouvrir le fichier",
   "Auth_Close": "Fermer",
   "Auth_Session": "Session",
-  "Auth_Session_Name": "Nom de la session",
-  "Auth_Save_Session": "Sauvegarder la session",
-  "Auth_Change_Session": "Changer de session",
-  "Auth_Saved_Sessions": "Sessions enregistrées",
-  "Auth_Session_Name_Aleary_Exist": "Ce nom de session existe déjà",
-  "Auth_Create_Session": "Créer une session",
-  "Auth_No_Active_Session": "Aucune session active",
-  "Auth_Choose_Session": "Choisir une session",
-  "Auth_Delete_Session": "Supprimer la session",
+  "Auth_SessionName": "Nom de la session",
+  "Auth_SaveSession": "Sauvegarder la session",
+  "Auth_ChangeSession": "Changer de session",
+  "Auth_SavedSessions": "Sessions enregistrées",
+  "Auth_SessionNameAlreadyExist": "Ce nom de session existe déjà",
+  "Auth_CreateSession": "Créer une session",
+  "Auth_EditSession": "Editer la session",
+  "Auth_AlwaysActive": "Toujours active ?",
```

Comment:
Auth_Session, Auth_SavedSessions, Auth_SaveSession, Auth_ChangeSession not used anymore